### DISCUSSION TITLE
when ColumnDisplayingChanged  will be available?

### DISCUSSION THREAD

**Discussion by danik121** [Megabit/Blazorise#5603] [Q&A] [answered]
when ColumnDisplayingChanged  will be available in datagrid?

**Comment by David-Moreira** [comment, marked as answer]
Hello,
it's coming in v1.6 I think we are nearing release. @stsrki could you update us on release ETA?

In the meantime you can use one of our dev versions to test the feature ahead of time and let us know if you have any feedback.
https://www.myget.org/gallery/blazorise
